specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2023-04-15-preview/images.json

```diff
@@ -0,0 +1,543 @@
+        "operationId": "Images_Generate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionQueryParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The specification of the images that should be generated.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GenerateImagesCreation"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Created",
+            "headers": {
+              "operation-location": {
+                "type": "string",
+                "format": "url",
+                "description": "URL to poll for the operation result."
+              },
+              "x-ms-request-id": {
+                "type": "string",
+                "format": "uuid",
+                "description": "Service generated correlation id identifying the request, in the form of a GUID."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/GenerateImagesResponse"
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code indicating what went wrong."
+              },
+              "x-ms-request-id": {
+                "type": "string",
+                "format": "uuid",
+                "description": "Service generated correlation id identifying the request, in the form of a GUID."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "Creating an image generation operation.": {
+            "$ref": "./examples/post_images_generate.json"
+          }
+        }
+      }
+    },
+    "/operations/images/{operation-id}": {
+      "get": {
+        "summary": "Returns the status of the operation",
+        "operationId": "Operations_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionQueryParameter"
+          },
+          {
+            "in": "path",
+            "name": "operation-id",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "description": "The id of the operation",
+            "maxLength": 36,
+            "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$"
```

Comment:
>"pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$" [](http://example.com/codeflow?start=12&length=90)

other properties do not have a pattern. See x-ms-request-id. Should we add it everywhere, or remove here?

Reply:
Removed it as it is not necessarily needed